Automattic/newspack-plugin
# Relocate Starter Content Newspack Logo

### All Submissions:






### Changes proposed in this Pull Request:

Starter Content installs the Newspack logo in the media library and uses it as the initial logo. Recently the `assets` directory was removed from releases. The logo asset had been in this directory, therefore release builds could not successfully create the asset. The symptom was a massively tall header, because the logo asset is not available. 

### How to test the changes in this Pull Request:

1. Check out this branch, create a release build.
2. Unzip the archive, verify that it contains `includes/raw_assets/images/newspack-logo.png`
3. Spin up a Jurassic Ninja instance. Install the release build.
4. Step through Setup, including Starter Content. 
5. Verify that the logo is in the media library and has been added to the theme.

### Other information:
